code/API_definitions/device-reachability-status-subscriptions.yaml

```diff
@@ -44,6 +44,26 @@ info:
       - the Access Token `sinkCredential` (optionally set by the requester) expiration time has been reached
       - the API server has to stop sending notification prematurely
 
+    **Note on combined usage of ``initialEvent`` and ``subscriptionMaxEvents``**:
+
+    If an event is triggered following ``initialEvent`` set to true,
+    this event will be counted towards ``subscriptionMaxEvents`` (if provided).
+
+
+    **Clarification on ``initialEvent`` & ``event-type`` behaviour:**
+
+    Following table illustrate behaviour regarding event triggering depending on **initial** reachability state of the device:
+
+    | subscrived event-type | device reachability status at subscription time | event send if ``initialEvent`` set to true |
```

Comment:
minor typos in table header:
subscrived --> subscribed
event send --> event sent